irohad/model/execution/impl/command_executor.cpp

```diff
@@ -364,7 +371,7 @@ namespace iroha {
           // Account must be well-formed (no system symbols)
           std::all_of(std::begin(create_account.account_name),
                       std::end(create_account.account_name),
-                      [](char c) { return std::isalnum(c); });
+                      [](char c) { return std::isalnum(c) and islower(c); });
```

Comment:
when we create account is should be in lower case, yes.
We should also support this everywhere. 
So in every place where account_id is passed as an argument, we should check if it is lower case.